BLE Indoor Positioning/src/main/java/com/nexenio/bleindoorpositioning/ble/beacon/signal/WindowFilter.java

```diff
@@ -0,0 +1,52 @@
+package com.nexenio.bleindoorpositioning.ble.beacon.signal;
+
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Created by leon on 09.01.18.
+ */
+
+public abstract class WindowFilter implements RssiFilter {
+
+    protected static final long DURATION_DEFAULT = TimeUnit.SECONDS.toMillis(3);
+
+    protected long duration = DURATION_DEFAULT;
+    protected long maximumTimestamp = System.currentTimeMillis();
+    protected long minimumTimestamp = maximumTimestamp - DURATION_DEFAULT;
```

Comment:
No need to set this here, if the only available constructor will override the variable anyway.